ui/src/app/utils/aou_external_links.tsx

```diff
@@ -2,3 +2,8 @@ export const CROMWELL_INTRO_LINK =
   'https://cromwell.readthedocs.io/en/stable/tutorials/FiveMinuteIntro/';
 export const WORKFLOW_AND_WDL_LINK =
   'https://support.terra.bio/hc/en-us/articles/360037117492-Overview-Getting-started-with-WDL';
+
+export const SAS_INTRO_LINK =
```

Comment:
I think that these urls would be better placed in:
https://github.com/all-of-us/workbench/blob/046ad35626405c4df1f7683d3fc8cb61b76f6c60/ui/src/app/utils/zendesk.ts

Reply:
Yeah thats what i was thinking but the urls doesnt seem to have zendesk pattern, will follow up  offline